dev-packages/e2e-tests/test-applications/default-browser/src/index.js

```diff
@@ -0,0 +1,14 @@
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: process.env.E2E_TEST_DSN,
+  integrations: [Sentry.browserTracingIntegration()],
+  tracesSampleRate: 1.0,
+  release: 'e2e-test',
+  environment: 'qa',
```

Comment:
just FYI, you couldn't have known this: setting the environment to `'qa'` is something we do that, if we actually want to send events from this app to Sentry, we can make sure it arrives and isn't dynamically sampled/dropped by the Sentry backend.